docs/internal/types.md

```diff
@@ -486,14 +486,29 @@ interface Addable
 {
   +(Addable & readonly, Addable & readonly) : Addable & imm;
 }
-class Foo : Addable { ... }
-class Bar : Addable { ... }
-class Baz : Addable { ... }
+class Foo : Addable
+{
+  // Implements interface and operates on Foo only (as expected)
```

Comment:
It does not implement the interface.  This method does not have the signature from the interface.